cypress/integration/helloworld_test.ts

```diff
@@ -0,0 +1,122 @@
+describe('Hello world e2e', function () {
+    const newAppName = `e2e-HelloWorld-${new Date().getTime() - 1528151000000}`
+    const username = `UserName`
+    const action01 = 'Hello World!'
+    /**
+     * FEATURE: New Model
+     */
+    it('CREATE NEW MODEL with random name and verify name on application page', function () {
+      // Open application
+      cy.visit('http://localhost:5050')
+  
+      // Click the button to create app
+      cy
+        .get('[data-testid="apps-list-button-create-new"]')
+        .click()
+  
+      // Ensure that name input is focused
+      cy.focused()
+  
+      cy
+        .get('[data-testid="app-create-input-name"]')
+        .type(newAppName)
+  
+      // Click the submit button
+      cy
+        .get('[data-testid="app-create-button-submit"]')
+        .click()
+  
+      // Ensure app page displays new application title
+      cy
+        .get('[data-testid="app-index-title"]')
+        .should(el => {
+          expect(el).to.contain(newAppName)
+        })
+    })
+  
+    /**
+     * FEATURE:  New Identity
+     */
+    it('it should create a NEW ENTITY', () => {
+      // Click entities navigation tab
+      cy.get('a[href$="/entities"]')
+        .click()
+  
+      // Click new entity button
+      cy.get('[data-testid="entities-button-create"]')
+        .click()
+  
+      // Enter name for entity
+      cy.get('[data-testid="entity-creator-input-name"]')
+        .type(username)
+  
+      // Select multi-value
+      cy.get('[data-testid="entity-creator-input-multivalue"]')
+        .click()
+  
+      // Select the submit button
+      cy.get('[data-testid="entity-creator-button-save"]')
+        .click()
+  
+      // Ensure new row is added to entities list with new entity name
+      cy.get('.ms-DetailsRow-cell')
+        .contains(username)
+    })
+  
+    /**
+     * FEATURE: New Action
+     */
+    it('it should be able to ADD AN ACTION', () => {
+  
+      // Click actions navigation tab
+      cy
+        .get('a[href$="/actions"]')
+        .click()
+  
+      // Click new action button
+      cy
+        .get('[data-testid="actions-button-create"]')
+        .click()
+  
+      // Click Action Type dropdown
+      cy
+        .get('[data-testid="dropdown-action-type"]')
+        .click()
+        .click()
+  
+      // Enter action response phrase: Hello World!
+      cy
+        .get('div[data-slate-editor="true"]')
```

Comment:
This should be more specific to ensure you get the correct element in cases where the are multiple.  Also, would prefer not to couple selector to implementation detail of slate.  In other words, do not rely on that attribute.  Should only know about `contenteditable`